src/main/resources/config/automation_SCPAPI_QA.properties

```diff
@@ -42,3 +42,8 @@ scp.delete.entitytype.endpoint=/scp/v1/entity-types/delete/
 scp.getreviewer.id=Content_reviewer@guerrillamail.info
 scp.getreviewer.pass=Password@123
 scp.getreviewer.list.verify=all_rights@yopmail.com
+scp.create.project.endpoint=/scp/v1/projects/update
+scp.update.project.endpoint=/scp/v1/projects/update/
+scp.delete.project.endpoint=/scp/v1/projects/update
+scp.qa.projectdetails=/scp/v1/projects/details/
+scp.project.details.invalidId = 9999999
```

Comment:
why we need a key for invalidID ..can use any random one?

Reply:
Done